apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/AktivitetspliktOppgaveService.kt

```diff
@@ -78,12 +97,14 @@ data class AktivitetspliktInformasjonBrevdataRequest(
             utbetaling = this.utbetaling,
             redusertEtterInntekt = this.redusertEtterInntekt,
             skalSendeBrev = this.skalSendeBrev,
+            // TODO: Hvor skal vi hente brevId fra her? db?
```

Comment:
Ja, jeg tenker det er greit å bare hente denne i samme sleng som de andre feltene fra DB

Reply:
Foreldet kommentar, denne er kun for å lagre brevutfall.